### DISCUSSION TITLE
how to install phoebe

### DISCUSSION THREAD

**Discussion by RongkunChen** [phoebe-team/phoebe#185] [Build help] [answered]
I tried to install phoebe, but I couldn't. This is an error warning I encountered during installation and I am not sure how to resolve it. thank you.

![1684995748245](https://github.com/mir-group/phoebe/assets/95059232/c8a294b6-629d-454a-a9e1-e917391b7b08)
![1684995846916](https://github.com/mir-group/phoebe/assets/95059232/5235a261-6b39-4aab-9731-7e488e326f34)

**Comment by jcoulter12** [comment]
Hi @xuemoyoushang,

Thanks for your interest in Phoebe! Let’s see if we can help you to get the code built.

I have some suspicions regarding what might be wrong with your build, but need more information to help.

Can you provide:
1) the steps you took to build the code
2) the output of CMake, as in what it prints on the commandline.
3) any edits you made to CmakeLists.txt or otherwise.
4) details about your environment (like your OS, which compilers you’re trying to use, etc) 

Thanks!
Jenny Coulter

**Comment by RongkunChen** [comment]
Hi&nbsp;Jenny Coulter，
Thanks for your replay！
1.&nbsp; Installed gcc-9.2.0, intel-oneapi, git-2.36.1, hdf5-1.14.0.
2.&nbsp; Don't have modified CmakeLists.txt
3.&nbsp; Cmake command :cmake ..&nbsp; &nbsp;-DCMAKE_C_COMPILER=/home/software/gcc/gcc-9.2.0/bin/gcc -DCMAKE_CXX_COMPILER=/home/software/gcc/gcc-9.2.0/bin/g++ and make -j $8






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mir-group/phoebe"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月25日(星期四) 晚上8:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mir-group/phoebe] how to install phoebe (Discussion #185)





 
Hi @xuemoyoushang,
 
Thanks for your interest in Phoebe! Let’s see if we can help you to get the code built.
 
I have some suspicions regarding what might be wrong with your build, but need more information to help.
 
Can you provide:
  
the steps you took to build the code
 
the output of CMake, as in what it prints on the commandline.
 
any edits you made to CmakeLists.txt or otherwise.
 
details about your environment (like your OS, which compilers you’re trying to use, etc)
  
Thanks!
 Jenny Coulter

**Comment by RongkunChen** [comment]
[command.docx](https://github.com/mir-group/phoebe/files/11565128/command.docx)

**Comment by jcoulter12** [comment, marked as answer]
Hi @xuemoyoushang, 

This document you sent has some warnings that it's corrupted or contains material that shouldn't be in a word document.

Could you possible paste just the output of cmake (not the outptut of build, just cmake) here? It's a relatively short amount of information. 
Because you're specifying these compilers as arguments, I'm pretty sure we'll find the issue from the output of cmake.

Again also, what is your operating system? Is this ubuntu? 
Another question -- did you set up parallel hdf5, or is are you planning to use Phoebe with serial HDF5? 

Thanks, 
Jenny
